frontend/src/components/Playback/OsdPlayer.vue

```diff
@@ -0,0 +1,336 @@
+<template>
+  <v-overlay
+    v-model="overlay"
+    contained
+    scrim="transparent"
+    width="100%"
+    height="100%">
+    <div
+      class="d-flex flex-column justify-space-between align-center player-overlay">
+      <div class="osd-top pt-s pl-s pr-s">
```

Comment:
I believe we can use https://vuetifyjs.com/en/components/overlays/ instead of our custom overlay?